src/Raven.Studio/typescript/common/shell/accessManager.ts

```diff
@@ -96,9 +178,19 @@ class accessManager {
         showDatabaseRecordMenuItem: this.operatorAndAbove,
         showDatabaseIDsMenuItem: this.operatorAndAbove,
         showConnectionStringsMenuItem: this.operatorAndAbove,
-        enableConnectionStringsMenuItem: this.clusterAdminOrClusterNode,
-        enableConflictResolutionMenuItem: this.clusterAdminOrClusterNode
+        disableConnectionStringsMenuItem: this.disableIfNotAdminAccessPerDatabase,
+        disableConflictResolutionMenuItem: this.disableIfNotClusterAdminOrClusterNode
```

Comment:
we need cluster admin for doing that? It means cloud users can't never change that, right?

Reply:
done